content/altinn-studio/v9/getting-started/serviceowner/_index.nb.md

```
@@ -0,0 +1,32 @@
+---
+
+## Lag en bruker
+Alle som ønsker å teste og/eller bruke Altinn Studio må først lage en bruker. Dette er selvbetjent.
+[Følg denne veiledningen for å komme i gang](create-user). 
+
+Om du skal ha tilgang til å lage tjenester for en organisasjon må du kontakte den som administrerer Altinn Studio
+for din organisasjon.
+
+## For deg som skal lage en tjeneste
+1. [Lag et enkelt skjema](../development/basic-form)
+2. [Ta i bruk flere konsepter i skjema]()
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)